common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -88,10 +88,10 @@
                 </div>
             }
 
-            case Some(media@InlineYouTubeMediaAtom(_)) => {
+            case Some(media@InlineYouTubeMediaAtom(_)) if item.cardTypes.showYouTubeMediaAtomPlayer => {
```

Comment:
Yes, that would be possible but we'd need to design that template, so I thought it better to do that in a subsequent PR.